### PR TITLE
Link custom place holder

### PR DESCRIPTION
**Pull request by Sapir-Shamay**

### CODE REVIEW

File: docs/UiSettings.md [hunk 1]
```diff
@@ -23,3 +23,10 @@ The `uiSettings` object exposes the following properties:
   This function determines the 'Add a nofollow tag' link panel checkbox visiblity.
 
   By default, its predicate is `relValue !== 'nofollow'`
+
+- `placeHolder` : 
+
+  This variable is an url place holder of type string
```
Comment (mharsat) [inline docs/UiSettings.md]:
```suggestion
  This variable determines the placeholder to use for the link panel.
```

File: docs/UiSettings.md [hunk 2]
```diff
@@ -23,3 +23,10 @@ The `uiSettings` object exposes the following properties:
   This function determines the 'Add a nofollow tag' link panel checkbox visiblity.
 
   By default, its predicate is `relValue !== 'nofollow'`
+
+- `placeHolder` :
```
Comment (mharsat) [inline docs/UiSettings.md]:
```suggestion
- `placeholder` : `string`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,6 +18,8 @@
     - [#519](https://github.com/wix-incubator/rich-content/pull/519) add props: topMargin, backgroundColor, foregroundColor. Remove counter when only 1 image. Larger z-index
   - `editor-common` 
     - [#526](https://github.com/wix-incubator/rich-content/pull/526) `onAtomicBlockFocus` is invoked with `undefined` when an atomic block loses focus
+  - `link place holder`
```
Comment (mharsat) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -18,6 +18,8 @@
...
+    - [#530](https://github.com/wix-incubator/rich-content/pull/530) add customized link place holder
```
Comment (mharsat) [inline CHANGELOG.md]:
```suggestion
    - [#530](https://github.com/wix-incubator/rich-content/pull/530)`uiSettings.linkPanel.placeholder` determines the link panel's placeholder
```

File: packages/editor-common/web/src/Components/LinkPanel.jsx [hunk 5]
```diff
@@ -172,5 +174,6 @@ LinkPanel.propTypes = {
   dropDown: PropTypes.object,
   onEnter: PropTypes.func,
   onEscape: PropTypes.func,
+  placeHolder: PropTypes.string,
```
Comment (mharsat) [inline packages/editor-common/web/src/Components/LinkPanel.jsx]:
```suggestion
  placeholder: PropTypes.string,
```

File: packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js [hunk 6]
```diff
@@ -87,7 +87,8 @@ class ImageSettings extends Component {
     } = this.props;
 
     const { linkPanel } = uiSettings || {};
-    const { blankTargetToggleVisibilityFn, nofollowRelToggleVisibilityFn } = linkPanel || {};
+    const { blankTargetToggleVisibilityFn, nofollowRelToggleVisibilityFn, placeHolder } =
```
Comment (mharsat) [inline packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js]:
```suggestion
    const { blankTargetToggleVisibilityFn, nofollowRelToggleVisibilityFn, placeholder } =
```

File: packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js [hunk 7]
```diff
@@ -228,6 +229,7 @@ class ImageSettings extends Component {
                     theme={theme}
                     t={t}
                     ariaProps={{ 'aria-labelledby': 'gallery_image_link_lbl' }}
+                    placeHolder={placeHolder}
```
Comment (mharsat) [inline packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js]:
```suggestion
                    placeholder={placeholder}
```

File: packages/plugin-image/web/src/toolbar/image-settings.js [hunk 8]
```diff
@@ -116,7 +116,8 @@ class ImageSettings extends Component {
     const { src, metadata = {} } = this.state;
 
     const { linkPanel } = uiSettings || {};
-    const { blankTargetToggleVisibilityFn, nofollowRelToggleVisibilityFn } = linkPanel || {};
+    const { blankTargetToggleVisibilityFn, nofollowRelToggleVisibilityFn, placeHolder } =
```
Comment (mharsat) [inline packages/plugin-image/web/src/toolbar/image-settings.js]:
```suggestion
    const { blankTargetToggleVisibilityFn, nofollowRelToggleVisibilityFn, placeholder } =
```

File: packages/plugin-image/web/src/toolbar/image-settings.js [hunk 9]
```diff
@@ -213,6 +214,7 @@ class ImageSettings extends Component {
               t={t}
               ariaProps={{ 'aria-labelledby': 'image_settings_link_lbl' }}
               languageDir={languageDir}
+              placeHolder={placeHolder}
```
Comment (mharsat) [inline packages/plugin-image/web/src/toolbar/image-settings.js]:
```suggestion
              placeholder={placeholder}
```

**Comment by shaulgo** [comment]
Is there a reason this a prop and not from translations?

**Pull request merged**